bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/CallOptionsFactory.java

```diff
@@ -39,7 +39,23 @@
    * contain state, specifically start time with an expiration is set; in cases when timeouts are
    * used, implementations should create a new CallOptions each time this method is called.
    */
-  <RequestT> CallOptions create(MethodDescriptor<RequestT, ?> descriptor, RequestT request);
+  <RequestT> CallOptions createForRpc(RequestT request);
```

Comment:
Note the renaming of the original create() method - now named as createForRpc and createForRpcAttempt to be more explicit

Reply:
Can you also update the javadoc that despite the name being RPC, its actually used for operations?

Reply:
I had already renamed the method to createForOperation, though I may still do the refactorings we discussed offline.